docs/DeveloperGuide.md

```diff
@@ -448,6 +479,25 @@ As the users of this application would be hotel mangers of small hotels, there w
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Summary of commands
+
+This is a quick overview of all the commands and their functionalities.
+
+| Action                 | Format                                                                                                                    | Examples                                                                                                                                              |
+|------------------------|---------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**                | `add n/NAME p/PHONE e/EMAIL rm/ROOM dr/DATE_RANGE ng/NUMBER_OF_GUESTS [rq/REQUEST]`                                       | `add n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/19/05/20 - 24/05/22 ng/3` <br> Adds guest John Doe to GuestBook with his relevant details. |
+| **Bill**               | `bill INDEX b/BILL`                                                                                                       | `bill 2 b/99.99` <br> Increases the bill of guest at second index by 99.99.                                                                           |
```

Comment:
Should we include the sign after the b/? or like have another example for deduct?

Reply:
Will include.